src/schema/notifications.ts

```diff
@@ -98,13 +126,15 @@ export class ZimbraNotifications {
 				const id = findDataId(this.cache, '$ROOT_QUERY.search', dataId =>
 					r.test(dataId)
 				);
+				const { sortBy }: any = getVariablesFromDataId(id) || {};
```

Comment:
sortBy always contains strings, any reason to give {} as default value?

Reply:
getVariablesFromDataId(id) this function returns the json object of provide id and from that json we are destructing sortBy while adding item into list.
But while deleting item from list, we get don't get any "id" so function will return 'undefined' so to handle that i'm doing this. as per discussed with santosh